README.md

```diff
@@ -117,8 +117,8 @@ Hurray! You successfully made a contribution!
 
 ### **Contents**
 
-|  |         |      |      |     |
-| ---------| ------- | ---- | ---- | --- |
+|          |         |      |      |     |          |
+| ---------| ------- | ---- | ---- | --- | -------- |
```

Comment:
Seems like you have added an extra, unnecessary column. Why is that?

Reply:
https://github.com/EddieHubCommunity/hacktoberfest-practice/pull/470#discussion_r704717691

Reply:
> 
> 
> Seems like you have added an extra, unnecessary column. Why is that?

@adityaraute Without it, it would break the table. thanks.

Reply:
I am talking about the last column, not the first row. It worked well before, didn't it?